tools/chunk_graph.py

```diff
@@ -37,9 +40,9 @@ def _chunk_graph(g, name, ndata_paths, edata_paths, num_chunks, output_path):
     ):
         edata_paths = {g.etypes[0]: ndata_paths}
     # Then convert all edge types to canonical edge types
-    etypestrs = {etype: ":".join(etype) for etype in g.canonical_etypes}
+    etypestrs = {etype: _etype_tuple_to_str(etype) for etype in g.canonical_etypes}
```

Comment:
I will revert this change for two reasons:
(1) Scripts under tools should try to avoid calling internal DGL utilities. Think of them as user-written scripts.
(2) The chunked graph format is different from graph partition book, though they happen to represent canonical edge type in the same way.